doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -150,6 +150,8 @@ $ docker ps
 CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
 ```
 
+Support for NGSIv2 notifications it's added from version above 1.17.0. For this pourpose it's been added a new dir [/NGSIv2](cygnus-ngsi/resources/ngsi-examples/NGSIv2) wich contans script files in order to emulate som NGSIv2 notification types. 
```

Comment:
```suggestion
Support for NGSIv2 notifications has been added from version above 1.17.1. For this purpose, it's been added a new dir [/NGSIv2](cygnus-ngsi/resources/ngsi-examples/NGSIv2) which contains script files in order to emulate some NGSIv2 notification types. 
```

Reply:
Fixed in [3e19cae](https://github.com/telefonicaid/fiware-cygnus/pull/1762/commits/3e19cae79a547c62d62434cdf9e841b40ede85b3)